I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 2889, in script
    e "왁? 깜짝 놀랐네" with vp
  File "game/script.rpy", line 2889, in <module>
    e "왁? 깜짝 놀랐네" with vp
NameError: name 'vp' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 2889, in script
    e "왁? 깜짝 놀랐네" with vp
  File "C:\Users\cante\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 920, in execute
    say_menu_with(self.with_, renpy.game.interface.set_transition)
  File "C:\Users\cante\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 739, in say_menu_with
    what = renpy.python.py_eval(expression)
  File "C:\Users\cante\Desktop\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\cante\Desktop\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 2889, in <module>
    e "왁? 깜짝 놀랐네" with vp
NameError: name 'vp' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.3.22090809
디지몬 시커즈 1.02
Mon Apr 17 11:50:17 2023
